frontend/public/components/search.tsx

```diff
@@ -147,8 +146,15 @@ const SearchPage_: React.FC<SearchProps> = (props) => {
   };
 
   const getToggleText = (item: string) => {
-    const { labelPlural } = modelFor(item);
-    return labelPlural;
+    const { labelPlural, apiVersion, apiGroup } = modelFor(item);
+    return (
+      <span className="co-search-group__accordian-label">
+        {labelPlural}{' '}
+        <span className="text-muted co-truncate show co-nowrap small">
```

Comment:
Rather than showing this for every resource, does it make sense to follow what we do in the dropdown and only show it if there is more than one instance of a resource?

![Screen Shot 2020-02-13 at 2 43 23 PM](https://user-images.githubusercontent.com/895728/74472026-3a9e2300-4e6f-11ea-9f77-9b3d48853014.png)

I'm not sure truncation is the right solution here because you can lose the meaningful differentiator at mobile.
![localhost_9000_k8s_ns_robb_clusterserviceversions_orderBy=asc sortBy=Deployment(iPhone 5_SE) (1)](https://user-images.githubusercontent.com/895728/74472109-64574a00-4e6f-11ea-9627-4f134789353b.png)

And in this instance, you can't even see all of the resource name (which is a separate, but related bug).
![localhost_9000_k8s_ns_robb_clusterserviceversions_orderBy=asc sortBy=Deployment(iPhone 5_SE)](https://user-images.githubusercontent.com/895728/74472145-720ccf80-4e6f-11ea-9a46-05f9ecfc058a.png)

Reply:
@rhamilto - talked to @bmignano about the showing of the group and version for everything and she said it was fine.
I will remove the truncation in the next push.